api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -525,10 +525,6 @@ public Integer getId() {
 	}
 	
 	/**
-	 * TODO: the behavior of this method needs to be controlled by some sort of global property
-	 * because an implementation can define how they want their names to look (which fields to
-	 * show/hide)
-	 * 
```

Comment:
@robbyoconnor  Sir, I guessed given the update I made to the compareTo() method following the comparable class definition for compare(). This was no longer necessary?